app/com/lynxanalytics/sandbox/ScalaScript.scala

```diff
@@ -286,17 +286,22 @@ object ScalaScript {
     }
   }
 
+  private val findVariablesCache = new SoftHashMap[String, Set[String]]()                                             
```

Comment:
Is this thread-safe?

Reply:
Sorry for the stupid question. It is thread-safe.

Reply:
> Sorry for the stupid question. It is thread-safe.

Oh, thanks for checking! I just copied it from the other cached things in this file.